Tools/AP_Periph/AP_Periph.cpp

```diff
@@ -285,6 +285,10 @@ void AP_Periph_FW::init()
     notify.init();
 #endif
 
+#if AP_RELAY_ENABLED
```

Comment:
should use HAL_PERIPH_ENABLE_RELAY, can check that AP_RELAY_ENABLED is true

Reply:
Done, added errors if `AP_RELAY_ENABLED` is false and if `HAL_PERIPH_ENABLE_RELAY` and `HAL_PERIPH_ENABLE_PWM_HARDPOINT` is also defined since that is a conflicting use case of the same message.